components/Cards/docs/accessibility.md

```diff
@@ -0,0 +1,66 @@
+recommendations:
+
+#### Accessibility Labels for Nested Elements
+
+Set an appropriate[`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) 
+for meaningful content that is not textual by default, like images. Also make sure to enable the element's 
+[`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+myImageView.isAccessibilityElement = true
+myImageView.accessibilityLabel = "Image title or some info"
+```
+
+Objective-C
+```objc
+myImageView.isAccessibilityElement = YES;
+myImageView.accessibilityLabel = @"Image title or some info";
+```
+
+#### Accessibility Label for Tappable Cards
+
+Tappable or Selectable Cards are often used as items in a collection. In this mode, Cards often expose a single 
+viewable or actionable item to assistive technologies, by aggregating the cards content, instead of exposing their 
+nested elements individually. This mode works best when there is a limited number of nested elements that can 
+be described as a sentence. To enable this mode, set an appropriate [`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) value 
+for the Card, and make sure to update it each time the Card's content changes. Additionally, make sure to enable 
+the card's [`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement)
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+card.isAccessibilityElement = true
+card.accessibilityLabel = "Location \(locationLabel.accessibilityLabel) has 
+  \(amountLabel.accessibilityLabel) stars.";
+```
+
+Objective-C
+```objc
+	card.isAccessibilityElement = YES;
+	card.accessibilityLabel = [NSString stringWithFormat:@"Location %@ has %@ stars", 
+    locationLabel.accessibilityLabel, amountLabel.accessibilityLabel];
+```
+
+### Accessibility Actions
+
+#### Accessibility Actions for Tappable Cards with interactive elements
+
+In addition to aggregating the Card's label information, tappable Cards with actionable nested elements, like 
+buttons, sliders, and so on, should also aggregate the Cards actions into a single list. That is done by following the
+[`UIAccessibilityAction`](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilityaction) 
+guidelines or by adding  [`UIAccessibilityCustom
+Action`](https://developer.apple.com/documentation/uikit/uiaccessibilitycustomaction) to the Card's 
+[`accessibilityCustom
+Actions`](https://developer.apple.com/documentation/objectivec/nsobject/1615150-accessibilitycustomactions).
+
+For a specific example, check `CardsCollectionTintingExample` and `CardTintExampleCell` in the 
```

Comment:
I don't see any custom actions in these examples. Can you file an Issue for this and let someone come back to it once those changes have landed?

Reply:
I ended up moving it to a separate PR (https://github.com/material-components/material-components-ios/pull/4472/files).